google-cloud-storage/src/main/scala/org/apache/pekko/stream/connectors/googlecloud/storage/impl/Formats.scala

```diff
@@ -110,8 +110,8 @@ object Formats extends DefaultJsonProtocol {
       StorageObjectJson(readOnlyFields, writeableFields)
     }
     override def write(obj: StorageObjectJson): JsValue = {
-      val fields1 = obj.readable.toJson.asJsObject.fields
-      val fields2 = obj.writeable.toJson.asJsObject.fields
+      val fields1 = spray.json.enrichAny(obj.readable).toJson.asJsObject.fields
+      val fields2 = spray.json.enrichAny(obj.writeable).toJson.asJsObject.fields
```

Comment:
What's the error here?

Reply:
won't compile - it says that `.toJson` is not a method on the relevent classes. The implicit conversion `enrichAny` is not happening - so I have to do it explicitly.

Scala 3 compiler, unfortunately, still has some bugs.

Reply:
how about import the implicit conversation in scope explicitly，will that work？

Reply:
I tried basically everything. The code I have here is what the compiler generates in Scala 2 anyway.

Reply:
I'll try again

Reply:
wired, let's me try it too💖💖

Reply:
looks like you were right, sorting out the imports fixed the problem - strange how Scala 2 compiler finds the implicit much more easily than Scala compiler - but it's good to know that the Scala3 compiler doesn't have a bug